### ISSUE TITLE
[PHP 8.1] Warning when installing dependencies

### ISSUE THREAD

**Issue by EtienneM** [Scalingo/php-buildpack#228]
Here is an excerpt of a deployment logs:

```
-----> Installing application dependencies with Composer
PHP Deprecated: Return type of APCUIterator::current() should either be compatible with Iterator::current(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in Unknown on line 0
PHP Deprecated: Return type of APCUIterator::next() should either be compatible with Iterator::next(): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in Unknown on line 0
PHP Deprecated: Return type of APCUIterator::key() should either be compatible with Iterator::key(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in Unknown on line 0
PHP Deprecated: Return type of APCUIterator::valid() should either be compatible with Iterator::valid(): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in Unknown on line 0
PHP Deprecated: Return type of APCUIterator::rewind() should either be compatible with Iterator::rewind(): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in Unknown on line 0
Installing dependencies from lock file
```

I don't know if we can do something about it, or should just wait for a Composer update...

**Comment by EtienneM** [comment]
Unfortunately, fixing #226 does not fix this issue. I'm raising this issue to high because it's really an ugly error.

**Comment by EtienneM** [comment]
Releasing APCU 5.1.21 seems to be the way to go (https://github.com/krakjoe/apcu/releases/tag/v5.1.21):

> Fixed compatibility with PHP 8.1 by adding return types to `APCUIterator`.
